README.md

```diff
@@ -1,275 +1,98 @@
-	
-// Measure the most popular keywords used for different search operations in the app
-[[PiwikTracker sharedInstance] sendSearchWithKeyword:@"Galaxy" category:@"Books" numberOfHits:17];
-
-// Track goals and conversion rate
-[[PiwikTracker sharedInstance] sendGoalWithID:@"1" revenue:100];
-
-// Track outlinks to external websites and apps
-[[PiwikTracker sharedInstance] sendOutlink:@"anotherapp://somwhere/else?origin=myapp"];
-
-// Track downloaded files and content. Will show in a dedicated section in the Piwik Server
-[[PiwikTracker sharedInstance] sendDownload:@"htttp://someserver.com/image.png"];
-
-// Track ecommerce transactions
-PiwikTransaction *transaction = [PiwikTransaction transactionWithBuilder:^(PiwikTransactionBuilder *builder) {
-  builder.identifier =
-  [builder addItemWithSku: ... ]
-  ...
-  }];
-[[PiwikTracker sharedInstance] sendTransaction:transaction];
-
-// Track campaigns
-campaignURLString = ...
-[[PiwikTracker sharedInstance] sendCampaign:(NSString*)campaignURLString;
-
-// Track content impressions and interactions with ads and banners
-// Track an impression when the ad is shown
-[[PiwikTracker sharedInstance] sendContentImpressionWithName:@"DN" piece:@"dn_image.png" taget:@"http://dn.se"];
-// Track an interaction when the user tap on the ad
-[[PiwikTracker sharedInstance] sendContentInteractionWithName:@"DN" piece:@"dn_image.png" taget:@"http://dn.se"];
-
-// Set a custom user agent profile in requests sent to the Piwik Server
-// This can be used to provide additional information about the users device
-id<PiwikDispatcher> dispatcher = [PiwikTracker sharedInstance].dispatcher;
-if ([dispatcher respondsToSelector:@selector(setUserAgent:)]) {
-  [dispatcher setUserAgent:@"My-User-Agent"];
-}
+pod 'PiwikTracker', branch: 'swift3'
 ```
-	  	
-Check out the full [API documentation](http://piwik.github.io/piwik-sdk-ios/docs/html/index.html) for additional methods and details.
-
-A more detailed [Getting started guide](https://github.com/piwik/piwik-sdk-ios/wiki/Getting-started-guide) can be found in the Wiki section.
 
-###User ID
+Then run `pod install`. In every file you want to use the PiwikTracker, don't forget to import the framwork with `import PiwikTracker`.
 
-Providing the tracker with a user ID lets you connect data collected from multiple devices and multiple browsers for the same user. 
+## Usage
+### Configuration
 
-A user ID is typically a non empty string such as username, email address or UUID that uniquely identify the user. The User ID must be the same for a given user across all her devices and browsers.
+Befor the first usage, the PiwikTracker has to be configured. This is best to be done in the `application(_:, didFinishLaunchingWithOptions:)` method in the `AppDelegate`.
 
-```objective-c
-[PiwikTracker sharedInstance].userID = @"mattias.levin@gmail.com"
+```
+Tracker.configureSharedInstance(withSiteID: "5", baseURL: URL(string: "http://piwik.org/piwik.php")!)
```

Comment:
Perhaps a more general path example like `http://path/to/your/piwik/instance/piwik.php`?